Stripe/PublicHeaders/STPPaymentIntentParams.h

```diff
@@ -90,6 +105,17 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, copy, nullable, readwrite) NSString *returnUrl __attribute__((deprecated("returnUrl has been renamed to returnURL", "returnURL")));
 
+
+/**
+ `@YES` to save this PaymentIntent’s Source to the associated Customer,
+ if the Source is not already attached.
+ 
+ This should be a boolean NSNumber, so that it can be `nil`
+ 
+ This property has been renamed to `savePaymentMethodToCustomer` and deprecated.
```

Comment:
is this accurate? If I set `sourceParams` but not `paymentMethodParams` and then set `savePaymentMethod` will it save it the source? Reading the change to the comment above that seems accurate, but I'm wondering if we should keep them separate? Did you have a particular reason for combining?

Reply:
I think it's accurate - "save_source_to_customer has been renamed to save_payment_method." 
 (https://stripe.com/docs/upgrades#2019-02-11)

If you're thinking that `savePaymentMethod` really seems to imply it's..for PaymentMethod, not Source, I agree though.  I think combining makes sense because the semantics are basically the same to me and it matches the single parameter that the API expects, but the name is confusing.

Reply:
Yes, it is accurate!

Reply:
okay sounds good! Thanks for the clarification